modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java

```diff
@@ -795,28 +794,32 @@ private boolean _hasValidAssetVocabularies(long scopeGroupId)
 			AssetVocabularyServiceUtil.getGroupVocabularies(
 				PortalUtil.getCurrentAndAncestorSiteGroupIds(scopeGroupId));
 
-		Stream<AssetVocabulary> stream = assetVocabularies.stream();
+		if (assetVocabularies.isEmpty()) {
```

Comment:
@slnn This empty check is redundant. The foreach loop just skips its body when the iterable is empty.